src/dynapi/SDL_dynapi_overrides.h

```diff
@@ -497,6 +497,7 @@
 #define SDL_GetVideoDriver SDL_GetVideoDriver_REAL
 #define SDL_VideoInit SDL_VideoInit_REAL
 #define SDL_VideoQuit SDL_VideoQuit_REAL
+#define SDL_FlashWindow SDL_FlashWindow_REAL
```

Comment:
Do not insert new apis: append after the last.
Running `gendynapi.pl` under `src/dynapi/` will do that for you.

Reply:
i dunno what is expected to happen, but i just did it by hand now, so better check

Reply:
Looks OK.